ClosedXML_Tests/Excel/Styles/ColorTests.cs

```diff
@@ -28,7 +28,7 @@ public void ColorNotEqualOperatorInPlace()
         [Test]
         public void ColorNamedVsHTML()
         {
-            Assert.IsTrue(XLColor.Black == XLColor.FromHtml("#000000"));
+            Assert.That(XLColor.Black, Is.EqualTo(XLColor.FromHtml("#FF000000")));
```

Comment:
is there an alpha channel?  why is there an alpha channel? what does FF alpha mean? transparent or non transparent?